bundles/http/Citrix/DocuSign/73f4e3b3-e2f3-43b3-9ce6-7f12da7b714c.js

```diff
@@ -479,4 +508,30 @@ async function deleteRecipient({ client, dataStore, integrationParameters, actio
         throw new Error(await response.text())
     }
     dataStore.deleteById('envelopes_recipients_signers', actionParameters.recipient_Id)
+}
+async function envelopeview({ client, dataStore, integrationParameters, actionParameters }) {
+    const ACCOUNT_ID = integrationParameters.account_Id;
+    const envelopeViewResponse = await client.fetch(`accounts/${ACCOUNT_ID}/envelopes/${actionParameters.envelope_Id}/views/recipient`,
+        {
+            method: "POST",
+            body: JSON.stringify(
+                {
+                    "userName": `${actionParameters.userName}`,
+                    "email": `${actionParameters.email}`,
+                    "returnUrl": 'https://docusign.github.io/returnUrl/',
+                    "authenticationMethod": "None"
+                })
+        })
+
+    const { responseBody, errorMessage, errorStatus } = await validateResponse(envelopeViewResponse)
+    if (errorStatus) {
+        console.log(JSON.stringify(responseBody), errorMessage)
+        throw new Error(errorMessage)
+    }
```

Comment:
You can move this `if` block into the `validateResponse()`.
Then you don't have to repeat it in all functions.
And you can then return from `validateResponse` only `responseBody`.

Reply:
Done the changes